hackathon_site/registration/models.py

```diff
@@ -117,27 +135,59 @@ class Application(models.Model):
 
     q1 = models.TextField(
         null=False,
-        help_text="Why do you want to participate in NewHacks?",
-        max_length=1000,
+        help_text="How many hackathons have you been to?",
+        choices=HACKATHON_NUMBER_CHOICES,
     )
+
     q2 = models.TextField(
         null=False,
-        help_text="Tell us about your experience with software or other topics relevant to NewHacks!",
+        help_text="If you’ve been to a hackathon, briefly tell us your experience. If not, describe what you expect to see and experience. (1000 char max)",
```

Comment:
Don't need to put the "(1000 char max)", a character counter is rendered as part of the field